client/profile-wizard/steps/usage-modal.js

```diff
@@ -6,30 +6,17 @@ import { Component } from '@wordpress/element';
 import { compose } from '@wordpress/compose';
 import { withDispatch, withSelect } from '@wordpress/data';
 import interpolateComponents from 'interpolate-components';
-import {
-	Button,
-	CheckboxControl,
-	FormToggle,
-	Modal,
-} from '@wordpress/components';
+import { Button, Modal } from '@wordpress/components';
 import { Link } from '@woocommerce/components';
 import { OPTIONS_STORE_NAME } from '@woocommerce/data';
 
 class UsageModal extends Component {
```

Comment:
I think it might be worthwhile to move this to `packages/components` to make it more reusable.

Reply:
As I can see all components in the `packages/components` are stateless and this one is pretty much connected to `@woocommerce/data`. We can split the presentation part and behavior and move only presentation part to the components but to be honest I don't see a lot of value in having this modal without logic.

What do you think?

Reply:
Agreed on its usefulness without being connected.  We do have another component `Plugins` that is both connected and exported, but we can leave this one as is for now until we need it elsewhere.

If we can destructure props and remove the bottom padding on those actions, I think this LGTM.

Reply:
Added props destructuring in https://github.com/woocommerce/woocommerce-admin/pull/5248/commits/30251836eecaa12ab08e3b608ad160e42d47314a.
Add updated padding (actually replaced with margin-top) in https://github.com/woocommerce/woocommerce-admin/pull/5248/commits/05b32368e29162ea30582651a79488f2b8f28334